src/svgast/ast.py

```diff
@@ -195,3 +181,34 @@ def __new__(cls, *args):
 
     def __str__(self):
         return ' '.join(map(str, self))
+
+
+class EncodingFileWrapper:
```

Comment:
codecs.open?

Reply:
I had a go with the codecs thing, I couldn't get it to go - do you have a working example?

Reply:
```
import codecs

with codecs.open('/tmp/deleteme', 'w', 'ascii') as f:
    f.write('bumble')
```